tests/lib/assertions/valid-for-testing-pc-22-from-20.json

```
@@ -0,0 +1,39 @@
+{
+    "type": "model",
+    "authority-id": "FpA2tgNE8bUrUKSOhE6SOpsMdlcEJkDL",
+    "series": "16",
+    "brand-id": "FpA2tgNE8bUrUKSOhE6SOpsMdlcEJkDL",
+    "model": "my-model",
+    "architecture": "amd64",
+    "timestamp": "2021-06-01T12:18:12+00:00",
+    "grade": "dangerous",
+    "base": "core22",
+    "serial-authority": ["generic"],
+    "snaps": [
+        {
+            "default-channel": "22/edge",
+            "id": "UqFziVZDHLSyO3TqSWgNBoAdHbLI4dAH",
+            "name": "pc",
+            "type": "gadget"
+        },
+        {
+            "default-channel": "22/edge",
+            "id": "pYVQrBcKmBa0mZ4CCN7ExT6jH8rY1hza",
+            "name": "pc-kernel",
+            "type": "kernel"
+        },
+        {
+            "default-channel": "latest/edge",
+            "id": "amcUKQILKXHHTlmSa7NMdnXSx02dNeeT",
+            "name": "core22",
+            "type": "base"
+        },
+        {
+            "default-channel": "latest/stable",
```

not that it really matters in the slightest, but shouldn't this be edge to match the other snaps ?

```suggestion
            "default-channel": "latest/edge",
```